impeller/renderer/backend/gles/reactor_gles.cc

```diff
@@ -233,6 +234,13 @@ bool ReactorGLES::ConsolidateHandles() {
 
 bool ReactorGLES::FlushOps() {
   TRACE_EVENT0("impeller", __FUNCTION__);
+
+#ifdef IMPELLER_DEBUG
```

Comment:
How about setting the proc tables entries to be no-ops in case IMPELLER_DEBUG is set? Prevents having to litter the code with ifdef `IMPELLER_DEBUG`.

Reply:
We do this for [extensions in the proc table](https://github.com/flutter/engine/blob/1656dc4a8c631b259f031ae349d6f47f8e38cf37/impeller/renderer/backend/gles/proc_table_gles.cc#L124). For instance, get the proc but reset it if the extension is absent. You can do the same for debug extensions, discard ifndef IMPELLER_DEBUG.